package.json

```diff
@@ -5,7 +5,8 @@
     "build": "next build",
     "start": "next start",
     "lint": "next lint",
-    "test": "jest"
+    "test": "jest",
+    "prepare": "husky && husky install"
```

Comment:
I think husky installed was removed in husky 9: https://github.com/typicode/husky/releases/tag/v9.0.1